src/jdk.incubator.foreign/share/classes/jdk/internal/foreign/abi/SharedUtils.java

```diff
@@ -415,6 +416,15 @@ static MethodHandle wrapWithAllocator(MethodHandle specializedHandle,
     private static class AllocHolder {
 
         private static final CLinker linker = getSystemLinker();
+        // FIXME: This should go away. This is temporary hack to get testing on Windows going.
```

Comment:
Move the fixme closer to the hack, right before the `if` in the static block

Reply:
will do